### PR TITLE
add new page to the menu

### PR DESCRIPTION
**Pull request by HEskandari**
Related to https://github.com/Particular/docs.particular.net/pull/5423

### CODE REVIEW

**Comment by timbussmann** [comment]
same comment as on this PR: https://github.com/Particular/docs.particular.net/pull/5426#discussion_r690090545

**Review by mikeminutillo — APPROVED** [review]
Approved but there are 8 extra commits in here and I'm not sure why github is picking them up

File: menu/menu.yaml [hunk 1]
```diff
@@ -381,11 +381,11 @@
       Title: Microsoft Generic Host
     - Url: nservicebus/hosting/web-application
       Title: Web Application
-    - Url: nservicebus/hosting/azure-functions
-      Title: Azure Functions
+    - Url: nservicebus/hosting/azure-functions-service-bus
+      Title: Azure Functions ServiceBus
```
Comment (timbussmann) [inline menu/menu.yaml:385]: this should probably use `Service Bus`, not `ServiceBus`.

Maybe `Azure Functions (Service Bus)` or `Azure Functions for Service Bus`?

File: menu/menu.yaml [hunk 2]
```diff
@@ -381,11 +381,11 @@
...
       Articles:
-      - Url: nservicebus/hosting/azure-functions/service-bus
-        Title: Service Bus
+      - Url: nservicebus/hosting/azure-functions-service-bus/custom-triggers
+        Title: Custom Azure Function triggers
```
Comment (timbussmann) [inline menu/menu.yaml:388]: could be shortened given this is already under Azure Functions?

```suggestion
        Title: Custom triggers
```

File: menu/menu.yaml (hunk 1 above)
Comment (HEskandari) [reply menu/menu.yaml:385]:
```suggestion
      Title: Azure Functions for Service Bus
```

File: menu/menu.yaml [hunk 3]
```diff
@@ -381,11 +381,11 @@
       Title: Microsoft Generic Host
     - Url: nservicebus/hosting/web-application
       Title: Web Application
-    - Url: nservicebus/hosting/azure-functions
-      Title: Azure Functions
+    - Url: nservicebus/hosting/azure-functions-service-bus
+      Title: Azure Functions for Service Bus
```
Comment (timbussmann) [inline menu/menu.yaml:385]: ok, sorry for nitpicking, I just saw that the docspage uses "Azure Functions with Azure Service Bus" title, so maybe the alternative is this:

```suggestion
      Title: Azure Functions with Service Bus
```

I'll let you decide. PR is already approved, good with either option.

**Review by timbussmann — APPROVED** [review]

**Pull request merged**